src/main/java/net/sf/mzmine/modules/visualization/tic/TICPlot.java

```diff
@@ -681,7 +681,7 @@ public String generateLabel(final XYDataset xyDataSet,
 		}
 	    });
 
-	    numOfPeaks++;
+	   // numOfPeaks++;
```

Comment:
I agree - I was just not sure if there was a reason for increasing the variable at the specific position.